### DISCUSSION TITLE
CreateShipmentRequest -> Parameter ShippingServiceOfferID and ShippingServiceID

### DISCUSSION THREAD

**Discussion by Lemodo20** [abuzuhri/Amazon-SP-API-CSharp#534] [General]
Hi,

do use above function, you need thoose 2 parameters. For ShippingServiceID i found some list, but i can't find the ShippingServiceOfferID.

How can i get this value?

**Comment by Lemodo20** [comment]
nobody can help me?

**Comment by brm121966** [comment]
first you need to get a response from getEligibleShipmentServices()
